src/main/java/com/endava/soa_5/exception_and_io/SetProperty.java

```diff
@@ -0,0 +1,41 @@
+package com.endava.soa_5.exception_and_io;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class SetProperty {
+    private String result;
+    private InputStream inputStream;
+
+    public String getPropValues(String key) throws IOException {
+
+        try {
+            result = new String();
+            Properties properties = new Properties();
+            String fileProperties = "Resource.properties";
+            inputStream = getClass().getClassLoader().getResourceAsStream(fileProperties);
+
+            if (inputStream != null) {
```

Comment:
By implementing the previous comment - you may remove 20-24 lines, as you are sure that this property is initialized when the user is trying to make a new object of SetProperty class.

Reply:
Done!